src/api.rs

```diff
@@ -1612,7 +1641,9 @@ mod test {
     let limit = 10 - missing;
 
     // data[output_frameno] = (input_frameno, end_of_subgop)
-    let data = encode_frames(&mut ctx, limit, 0);
+    let data = encode_frames(ctx, limit, 0)
+      .map(|(fi, end_of_subgop)| (fi.input_frameno, end_of_subgop))
+      .collect::<Vec<_>>();
 
```

Comment:
might be nice for an assert on pyramid depth

Reply:
Done.